### PR TITLE
Create registration list

### PR DESCRIPTION
**Pull request by LarabfSantos**
I'm ready for this!

### CODE REVIEW

**Comment by Porim** [comment]
I've fixed some issues with this PR and added two files into .gitignore that shouldn't be pushed to GitHub, but I've no right to remove them. @J-Coke can you sort it out, please? Thanks.

Other than that I think this PR is ready to be merged.

**Review by J-Coke — APPROVED** [review]

File: cypress/e2e/registration-listspec.cy.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+describe('empty spec', () => {
```
Comment (Joseaie) [inline cypress/e2e/registration-listspec.cy.js:1]: Should this test be here?

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,5 @@
 node_modules
 cypress/screenshots
 cypress/videos
+.DS_Store
```
Comment (Joseaie) [inline .gitignore:4]: nit: these changes should be in a separate PR

File: cypress/e2e/registration-listspec.cy.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  it('passes', () => {
+    cy.visit('http://localhost:3000/registration-list')
+  })
+})
+
+describe('registration list page', () => {
+  it('displays the title', () => {
+    cy
+    .visit('http://localhost:3000/registration-list')
```
Comment (Joseaie) [inline cypress/e2e/registration-listspec.cy.js:10]: nit: To reduce repetition, you could put the `visit` in a `before` hook (https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests#Hooks)

File: cypress/e2e/registration-listspec.cy.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    .get('ul li:first')
+  })
+  it('displays last element of unordered list', () => {
+    cy
+    .visit('http://localhost:3000/registration-list')
+    .get('ul li:last')
+  })
+})
+
+describe('continue button', () => {
```
Comment (Joseaie) [inline cypress/e2e/registration-listspec.cy.js:25]: nit: Although `describe` and `context` blocks are essentially the same apparently, maybe this could go inside a `context` block within the `describe`. (Like in this example: https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests#Test-Structure)

**Comment by Joseaie** [comment]
Could you please add screenshots of what the page looks like as well as a description of the changes you made please?
